tensorflow/core/kernels/data_format_ops.cc

```diff
@@ -21,11 +21,11 @@ limitations under the License.
 
 #include <map>
 
-#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
 #include "tensorflow/core/framework/op_kernel.h"
 #include "tensorflow/core/framework/register_types.h"
 #include "tensorflow/core/framework/tensor.h"
 #include "tensorflow/core/platform/errors.h"
+#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
```

Comment:
Let's not move this import (although it won't follow the alphabetical ordering anymore). Internally, the import paths are slightly changed and the order before the change becomes alphabetical.

Reordering would cause issues on the import

Reply:
I see. Do people usually re-order the includes manually after clang-formatting or is there a setting I'm missing?

Reply:
Unfortunately this is a corner case that we haven't handled in particular.